Source/WebKit/WPEPlatform/wpe/WPEToplevel.cpp

```diff
@@ -154,6 +162,19 @@ static void wpe_toplevel_class_init(WPEToplevelClass* toplevelClass)
             WPE_TYPE_DISPLAY,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
 
+    /**
+     * WPEToplevel:max-views:
+     *
+     * The maximum number of #WPEView that can be added to the #WPEToplevel.
+     * A value of 0 means no limit.
+     */
```

Comment:
Shouln't this have a Since tag?

Reply:
No, because we haven't released this api yet.